addons/docs/docs/recipes.md

```diff
@@ -31,46 +31,50 @@ The only limitation is that your exported titles (CSF: `default.title`, MDX `Met
 
 Perhaps you want to write your stories in CSF, but document them in MDX? Here's how to do that:
 
-**Button.mdx**
+**Button.stories.js**
+
+```js
+import React from 'react';
+import { Button } from './Button';
+
+export default {
+  title: 'Demo/Button',
+  component: Button,
+  includeStories: [], // or simply don't load this file at all
```

Comment:
Should we allow `false` as a value here?

Reply:
Overloading the type is more trouble than it's worth IMHO

Reply:
OK, that's fair.